cas/cas.go

```diff
@@ -148,23 +148,23 @@ func (ds Store) ResolveKey(key string) (string, error) {
 		key = key[:lenKey]
 	}
 
-	cancel := make(chan struct{})
-	var k string
-	keyCount := 0
-	for k = range ds.stores[blobType].KeysPrefix(key, cancel) {
-		keyCount++
-		if keyCount > 1 {
-			close(cancel)
-			break
-		}
+	aciInfos := []*ACIInfo{}
+	if err := ds.db.Do(func(tx *sql.Tx) error {
```

Comment:
could you please move to splitting out the err assignment? I think it's slightly more readable in longer cases like this 

```
err := ...
if err != nil {
```